### PR TITLE
Milestone 3-Final  touches

### PR DESCRIPTION
**Pull request by eudondian**
- The look and feel of the application were improved by adding styles.
- The general layout of the wireframe was used as this was the only mandatory requirement.
- plain CSS  was used to add the style

### CODE REVIEW

File: src/modules/service.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+      },
+      body: JSON.stringify(newScoreObject),
+    },
+  ).then((res) => res.json().then((data) => data.result));
+  name.value = '';
+  score.value = '';
+};
+
+addInput.addEventListener('submit', addPost);
+
```
Comment (Deepakdanger) [inline src/modules/service.js:50]: - [ ] When trying to submit the score, I can't able to submit it to leaderboard API. Please fix it.

File: src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+}
+
+h1 {
+  text-align: center;
+  margin-top: 50px;
+  font-size: 50px;
+  font-weight: bolder;
+  color: rgb(235, 235, 235);
+  padding-top: 50px;
+}
```
Comment (Deepakdanger) [inline src/styles/style.css]: - [ ] Also, keep your design similar to the wireframe. The input form should be on the right.


### Your design
Screenshot from 2022-07-30 03-26-00

### Given design
Screenshot from 2022-07-30 03-24-45

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @eudondian, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @eudondian, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

**Pull request merged**